disable-breaking-updates.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+"""
+Disable breaking updates which will prompt users to download a deb or tar file
+and lock them out of Discord making the program unusable.
+
+This will dramatically improve the experience :
+
+ 1) The maintainer doesn't need to be worried at all times of an update which will break Discord.
+ 2) People will not be locked out of the program while the maintainer runs to update it.
+
+"""
+
+import json
+import os
+from pathlib import Path
+
+XDG_CONFIG_HOME = os.environ.get("XDG_CONFIG_HOME")
+if Path(f"{XDG_CONFIG_HOME}/discord/settings.json").is_file():
+    with open(f"{XDG_CONFIG_HOME}/discord/settings.json", "r+") as settings_file:
+        settings = json.load(settings_file)
+        if settings.get("SKIP_HOST_UPDATE"):
+            print("Disabling updates already done")
+        else:
+            skip_host_update = {"SKIP_HOST_UPDATE":True}
+            settings.update(skip_host_update)
+            settings_file.seek(0)
```

Comment:
I would just re-open the file in truncate mode

Reply:
Even better would be to overwrite it atomically. Unfortunately the Python standard library doesn't seem to have this, but the technique is basically:

* open a temp file in the same directory (it has to be the same directory otherwise `rename()` might not work)
* write the new content to the temp file
* `os.rename()` the temp file to overwrite the original

https://github.com/untitaker/python-atomicwrites is a comprehensive implementation of this, if it's easy to install third-party libraries.

https://www.edgewall.org/docs/tags-trac-0.11.7/epydoc/trac.util-pysrc.html#AtomicFile is a smaller implementation and probably more suitable for copy/paste.